src/install/kubernetes/guided/install-guided.mdx

```diff
@@ -0,0 +1,36 @@
+---
+componentType: default
+headingText: Install the Kubernetes integration
+---
+
+import kubernetesGuidedInstall from 'images/kubernetes_screenshot-crop_guided-install-ui.webp'
+
+import kubernetesGuidedInstallCommand from 'images/kubernetes_screenshot-crop_guided-install-ui-command.webp'
+
+
+
+The guided installation gives you control over the process, allowing you to choose which features to enable and which data to collect. It makes the installation process easier and simpler than the other ways to install the integration. It also offers a quickstart option that includes some optional, pre-built resources such as dashboards and alerts alongside the Kubernetes integration so that you can gain instant visibility into your Kubernetes clusters.
```

Comment:
The first two sentences are pretty clunky. Can you combine them and say: The guided install is the easiest and simplest way to install the integration while also providing the most control.

Reply:
Yes! Done!